src/main/java/com/es/phoneshop/model/cart/Cart.java

```diff
@@ -1,11 +1,15 @@
 package com.es.phoneshop.model.cart;
 
+import java.io.Serializable;
+import java.math.BigDecimal;
 import java.util.ArrayList;
 import java.util.List;
 
-public class Cart {
+public class Cart implements Serializable {
```

Comment:
where is the active link to cart? users need to have the ability to click smth to see the cart, they don't know urls